bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/zk/ZKMetadataDriverBase.java

```diff
@@ -260,6 +262,47 @@ public synchronized LedgerManagerFactory getLedgerManagerFactory()
         return lmFactory;
     }
 
+    public CompletableFuture<Void> disableHealthCheck() {
+        CompletableFuture<Void> createResult = new CompletableFuture<>();
+        try {
+            zk.create(getEnableHealthPath(), new byte[0], ZkUtils.getACLs(conf), CreateMode.PERSISTENT);
+            createResult.complete(null);
+        } catch (Exception e) {
+            createResult.completeExceptionally(e);
+        }
+        return createResult;
+    }
+
+    public CompletableFuture<Void>  enableHealthCheck() {
+        CompletableFuture<Void> deleteResult = new CompletableFuture<>();
+
+        try {
+            zk.delete(getEnableHealthPath(), -1);
+            deleteResult.complete(null);
+        } catch (Exception e) {
```

Comment:
KeeperException.NoNodeException is not a real error here (because it's already enabled)

Reply:
like this ?

    public CompletableFuture<Void> enableHealthCheck() {
        CompletableFuture<Void> deleteResult = new CompletableFuture<>();

        try {
            zk.delete(getEnableHealthPath(), -1);
            deleteResult.complete(null);
        } catch (KeeperException.NoNodeException noNodeException) {
            deleteResult.complete(null);
        } catch (Exception e) {
            deleteResult.completeExceptionally(e);
        }

        return deleteResult;
    }

Reply:
yes, we can add a debug log in that case

Reply:
OK ，I will fix